doc/source/citations.rst

```diff
@@ -52,12 +52,69 @@ TAD coordinates
 
 * The article containing the method used to call TAD boundaries: 
 
-  Wang, Y., Song, F., Zhang, B., Zhang, L., Xu, J., Kuang, D., Li, D., 
-  Choudhary, M.N.K., Li, Y., Hu, M., et al. (2018). 
+
+  Wang Y., Song F., Zhang B., Zhang L., Xu J., Kuang D., Li D., Choudhary M.N.K., 
```

Comment:
Do you think it will look better to put the articles as nested list?

Reply:
That is a very good point. done. thank you!